__tests__/support/setup/index.js

```diff
@@ -1,3 +1,5 @@
 import gql from '../gql';
+import smtp from '../smtp';
 
 global.gql = gql;
```

Comment:
The test should make use of the actual function, that is being used in ActionJS.